student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(classroom)
+    super
+    @classroom = classroom
+  end
```

Comment:
- [ ] The requirements requested you to extend the parent's constructor by adding `@classroom  `. Let's add other parameters here too. 
<img width="629" alt="Capture" src="https://user-images.githubusercontent.com/58504655/135908923-b2d77d01-c4e6-413d-b6ed-11f959ba74de.PNG">

Reply:
:heavy_check_mark: Fixed!